.jshintrc

```diff
@@ -27,17 +27,25 @@
 	"browser": true,
 
 	// Globals
```

Comment:
Dunno, maybe because JSON is not allowed to have comments and GitHub thinks this is plain JSON?